crates/aiken-project/src/blueprint/mod.rs

```diff
@@ -81,8 +81,8 @@ where
         &self,
         title: Option<&String>,
         purpose: Option<&Purpose>,
```

Comment:
I think the `purpose` can go completely now. We can identify validators just by their name / title.

Reply:
I left it so that it matches the spec update we discussed. would be nice to use this as a library later outside of aiken. but if the spec is going to remove it completely then sure.